src/console_cmds.cpp

```diff
@@ -859,7 +859,7 @@ DEF_CONSOLE_CMD(ConNetworkReconnect)
 			/* From a user pov 0 is a new company, internally it's different and all
 			 * companies are offset by one to ease up on users (eg companies 1-8 not 0-7) */
 			playas--;
-			if (playas < COMPANY_FIRST || playas >= MAX_COMPANIES) return false;
+			if (playas == COMPANY_FIRST || playas > MAX_COMPANIES) return false;
```

Comment:
Not equivalent?

Reply:
i meant to remove `playas--;` as well. the issue was COMPANY_FIRST is 0, and you can't have `< 0` for a byte...